mods/default/nodes.lua

```diff
@@ -1902,6 +1902,7 @@ function default.chest.register_chest(name, d)
 			local meta = minetest.get_meta(pos)
 			meta:set_string("infotext", "Locked Chest")
 			meta:set_string("owner", "")
+			meta:mark_as_private("owner")
```

Comment:
what's the point if the infotext reveals the owner anyway?

Reply:
Ok thanks, will edit.